dissect/target/tools/fs.py

```diff
@@ -106,16 +107,22 @@ def main():
 
     args = parser.parse_args()
 
+    if args.subcommand is None:
+        parser.error("No subcommand specified")
+
+    if not args.target.is_file():
+        parser.error(f"{args.target} is not a file or does not exist")
```

Comment:
As a suggestion to add a little more context:
```suggestion
        parser.error(f"Specified target '{args.target}' is not a file or does not exist")
```